src/Icon.js

```diff
@@ -26,7 +26,7 @@ StyledIcon.propTypes = {
   ...textAlign.propTypes
 };
 
-export const Icon = ({ name, theme, ...props }) => {
+const Icon = ({ name, theme, ...props }) => {
```

Comment:
Not quite the right line to pick but GitHub won't let me pick the one I actually want. I'm not sure I understand why Icon is exported withTheme(Icon) but in Input and Tooltip, withTheme was removed. Could use some help understanding that so I understand this PR better.

Reply:
Good question. The reason we're still using `withTheme` here is the line below this one. We are actually consuming the theme in this component as a way to customize the iconFontPrefix, so we need the provider to pass the theme into the Icon component.

As opposed to the other uses of `withTheme` which were band-aids to get theming working in the tests.